library/core/src/main/java/com/google/android/exoplayer2/text/subrip/SubripDecoder.java

```diff
@@ -95,8 +128,36 @@ protected SubripSubtitle decode(byte[] bytes, int length, boolean reset) {
         textBuilder.append(currentLine.trim());
       }
 
-      Spanned text = Html.fromHtml(textBuilder.toString());
-      cues.add(new Cue(text));
+      // Extract tags
+      SubtitleTagResult tagResult = extractTags(textBuilder);
```

Comment:
Given you wouldn't expect a tag to be separated over multiple lines, it seems cleaner to do this in the while loop above. You could replace the last line in the while loop with something like:
```
textBuilder.append(processLine(currentLine, tags));
```
Where `tags` is an `Arraylist` to which any new tags are added, and `processLine` is responsible for trimming the line and extracting tags. Note passing the list in as an argument also avoids having to define `SubtitleTagResult`.

You'd then make the `tags` list before the start of the while loop to pass in (you could even re-use the same single instance if you want, as is done for the `StringBuilder`, as long as you remember to clear it.